src/components/ProfileTopWifi.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div v-if="degree === 0"><img src="@/assets/Wifi2.svg" alt="wifi0"></div>
```

Comment:
v-if'i direkt `img` elementi uzerinde kullanabilirsiniz. ornek:
```suggestion
  <img v-if="degree === 0" src="@/assets/Wifi2.svg" alt="wifi0">
```
(alttaki satirlar icin de gecerli bu durum)